### ISSUE TITLE
Inserting into tables with uuid as the key type

### ISSUE THREAD

**Issue by Dizzzmas** [ArsenyYankovsky/typeorm-aurora-data-api-driver#48]
So I have a FK column with `uuid` type.

And when I'm trying to do:
```
entity.foreignId = "valid-uuid-formatted-string"
await repository.save(entity)
```

I'm getting this: 
```
"code": "BadRequestException",
    "message": "ERROR: column \"foreign_id\" is of type uuid but expression is of type character varying\n  Hint: You will need to rewrite or cast the expression.
```

Trying to rewrite it using `createQueryBuilder` and casts at the moment.

**Comment by mlg87** [comment]
@Dizzzmas did you ever figure this out? in the process of migrating our dbs to aurora serverless and this just started showing up
```
[GraphQLError: ERROR: operator does not exist: uuid = character varying
    Hint: No operator matches the given name and argument type(s). You might need to add explicit type casts.
    Position: 613] {
    locations: [ [Object] ],
    path: [ 'viewer' ],
    extensions: { code: 'INTERNAL_SERVER_ERROR', exception: [Object] }
  }
```
i assume it's the same issue youre running into as all of the queries work when run with a local db

**Comment by ArsenyYankovsky** [comment]
I've submitted a PR fixing this to the TypeORM repository [here](https://github.com/typeorm/typeorm/pull/7483) and released a new 2.0.0 version of the driver. 

While we're waiting for the TypeORM pull request to be merged and a new release, you can use [my fork of TypeORM](https://github.com/ArsenyYankovsky/typeorm), branch `data-api-prepare-hydrate`.

**Comment by ArsenyYankovsky** [comment]
You don't have to use my fork anymore since new TypeORM 0.3.32 was released. Feel free to comment here or open a new issue if something doesn't work.
